app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Apply for your visa to train to teach in England"
+description: |-
+  Learn more about types of visa non-UK citizens will need to study or work in England, and how to apply.
+related_content:
+    Career progression stories: "/blog/tag/career-progression"
+    Salaries and benefits of teaching in England : "/salaries-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+date: "2021-05-27"
+image: "static/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Overseas
+  - Foreign
+  - Non-UK
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+---
+
+If you’re applying for teacher training as a non-UK citizen, you will need a visa or immigration status allowing you to study or work in England. The information on this page is for:
+
+* non-UK citizens living in the UK
+* non-UK citizens living outside the UK
+* refugees or asylum seekers
+
+You will not need a visa to train to teach in the UK if you:
+
+* are an Irish citizen
+* have [settled or pre-settled status under the EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
+* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
+
+If you are currently studying in the UK on a student visa, you may be able to apply for a [graduate visa](#other-types-of-visa-you-can-use-to-work-and-study-in-england).
+
+### The EU Settlement Scheme 
+
+If you or your family are from the European Economic Area (EEA) or Switzerland, and you or a family member were living in the UK by 31 December 2020, you may still be able to [apply to the EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families).
```

Comment:
```suggestion
If you or your family are from the European Economic Area (EEA) or Switzerland, and you or a family member were living in the UK by 31 December 2020, you may still be able to apply to the EU Settlement Scheme.
```
again, just trying to remove the repetition of the same link with different link text

Reply:
They are the same now :)